GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -237,6 +237,14 @@ @interface GDTCORApplication ()
 
 @implementation GDTCORApplication
 
+/** A dispatch queue on which all task semaphore will populate and remove from
+ * sBackgroundIdentifierToSemaphoreMap.
+ */
+static dispatch_queue_t sSemaphoreQueue;
+
+/** An NSMutableDictionary for mapping backgroundIdentifier to task semaphore. */
```

Comment:
`/** For mapping backgroundIdentifier to task semaphore. */`

You don't need to reiterate the type in the comment.

Reply:
Changed